app/controllers/users_controller.rb

```diff
@@ -1,3 +1,29 @@
 class UsersController < ApplicationController
-  def new; end
+  def show
+    @user = User.find params[:id]
+    return if @user
+
+    flash[:danger] = t "Usernotfound"
```

Comment:
Tìm hiểu về I18n lazy lookup để define và sử dụng cho ngắn gọn em nhé
Chú ý cách định nghĩa, dùng snake case, fix tương tự bên dưới
```suggestion
    flash[:danger] = t ".user_not_found"
```